scripts/eth-allowed-packages.txt

```diff
@@ -10,6 +10,7 @@
 "github.com/ava-labs/libevm/common/math"
 "github.com/ava-labs/libevm/common/prque"
 "github.com/ava-labs/libevm/core/asm"
+"github.com/ava-labs/libevm/core/types"
```

Comment:
I think you can remove this if you apply the suggestion in lint_allowed_eth_imports.sh
Since we still have the types package, it's better to be explicit and require libevm imports as ethtypes.